cms/models/pluginmodel.py

```diff
@@ -250,6 +250,23 @@ def get_plugin_instance(self, admin=None):
         self._inst = instance
         return self._inst, plugin
 
+    def get_bound_plugin(self, admin=None):
```

Comment:
Why not makeing ``get_bound_plugin`` a wrapper to ``get_plugin_instance``:

    def get_bound_plugin(self, admin=None):
        return self.get_plugin_instance(admin)[0]